opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -31,6 +31,11 @@
      */
     Span activeSpan();
 
+    /**
+     * @return the active {@link SpanContext}. This is a shorthand for {@code Tracer.scopeManager().activeSpanContext()}.
+     */
+    SpanContext activeSpanContext();
```

Comment:
personally don't think these shortcuts should be here. just use the scope manager, and change docs of the activateSpan stuff to mention it internally uses the same. I don't know if Tracer.toSpan(SpanContext) has been done yet, but it would clear all of this up, possibly addressing @wu-sheng's gripe about not knowing when a span is made present on another thread.

Reply:
In the interest of scoping this pr (see what I did there?) I think it's better to move this discussion to a separate issue. I added this to be in line with the current conventions. I have not heard of `Tracer.toSpan(SpanContext)` yet. What are Wu's gripes you are referring to?

Reply:
this topic is indeed separately addressable, though should be done! I may have misunderstood your SELF link issue as the same end functionality as `Tracer.toSpan(SpanContext)`